packages/client/README.md

```diff
@@ -10,8 +10,6 @@ Firstly, install the package:
 
 ```bash
 npm install @vercel/client
-# or
-yarn add @vercel/client
```

Comment:
**question (non-blocking):** why was this removed?

Reply:
@styfle previous comment about our docs should either have yarn/npm/pnpm or just npm

Reply:
Probably because of my comment here https://github.com/vercel/vercel/pull/9198#discussion_r1066224797

In hindsight, we should leave revert the changes to `packages/*/README.md` because they are for the consumer and therefore not relevant to this PR. They can be updated in a future PR.